site/src/doc/tutorials/raster-reprojection.md

```diff
@@ -5,13 +5,24 @@ layout: default.hbs
 
 # Raster Reprojection
 
-OpenLayers has an ability to display raster data from WMS, WMTS, static images and many other sources in a different coordinate system than delivered from the server.
+OpenLayers can display raster data from WMS, WMTS, static images, and many other sources in a different coordinate system than delivered from the server. In cases where the source projection differs from the map view projection, source data can be reprojected on the client (in the browser).
+
+OpenLayers comes with built-in support for transforming coordinates (and reprojecting rasters) between a handful of projections or coordinate reference systems.
+
+The built-in reprojection support applies to the following projections:
+
+ * WGS 84 / Geographic (`EPSG:4326`)
+ * WGS 84 / Web or Spherical Mercator (`EPSG:3857`)
+ * WGS 84 / Universal Transverse Mercator (`EPSG:32601` through `EPSG:32660` and `EPSG:32701` through `EPSG:32760`)
```

Comment:
Since it not only affects raster reprojection, this should also be added to the class docs of `ol/proj/Projection`. It should also contain a note about the reduced accuracy, and that proj4js can be used to achieve higher accuracy if required.

Reply:
The docs now look like this
<img width="641" alt="image" src="https://github.com/user-attachments/assets/9f35ac02-0f4f-4039-979f-e739d06a3521">

Previously they looked like this
<img width="640" alt="image" src="https://github.com/user-attachments/assets/98d2610c-418a-4a9d-80b0-b67289a65ec5">